test/index.js

```diff
@@ -380,6 +382,24 @@ describe('HauteCouture', () => {
         expect(bigServer.match('get', '/arr-route-two')).to.exist();
     });
 
+    it('defines subscriptions in subscriptions/.', async () => {
+
+        const team = new Teamwork();
+        await bigServer.start();
+        const client = new Nes.Client(`ws://localhost:${ bigServer.info.port }`);
+        await client.connect();
+        await client.subscribe('/subscription-test', (data) => {
+
+            expect(data).to.equal({ id : 1, message: 'test' });
+            team.attend();
+        });
+
+        await bigServer.publish('/subscription-test', { id : 1, message: 'test' });
+        await team.work;
+        client.disconnect();
+        await bigServer.stop();
```

Comment:
I think it would be preferable to add `server.stop()` as a lab cleanup action (`flags.onCleanup = () => ...`) right after the server is started.